src/Mvc/Mvc.Core/ref/Microsoft.AspNetCore.Mvc.Core.netcoreapp3.0.cs

```diff
@@ -814,6 +814,11 @@ public partial interface IRequestFormLimitsPolicy : Microsoft.AspNetCore.Mvc.Fil
     public partial interface IRequestSizePolicy : Microsoft.AspNetCore.Mvc.Filters.IFilterMetadata
     {
     }
+    public partial class JsonOptions
+    {
+        public JsonOptions() { }
+        public System.Text.Json.Serialization.JsonSerializerOptions JsonSerializerOptions { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
```

Comment:
@pranavkm thoughts? Folwer and I like it because we get to write `.AddControllers().AddJsonOptions(...)` which feels a little better than putting it on mvc options

Reply:
Seems fine. It's closer to how you configure other formatters \ `AddXmlOptions`